mp4parse/src/lib.rs

```diff
@@ -40,15 +41,15 @@ const TABLE_SIZE_LIMIT: u32 = 30 * 60 * 60 * 24 * 7;
 
 /// A trait to indicate a type can be infallibly converted to `u64`.
 /// This should only be implemented for infallible conversions, so only unsigned types are valid.
-trait ToU64 {
-    fn to_u64(self) -> u64;
+trait IntoU64 {
+    fn into_u64(self) -> u64;
```

Comment:
Why the change here? If there was a clippy issue about it, I can't seem to reproduce that. If it's a question of rust conventions, I think "to" is more idiomatic than "into" based on [the rust API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv) for an owned -> owned conversion on `Copy` types.

Reply:
I was working around https://github.com/rust-lang/rust-clippy/issues/6727, but I'll revert the change and add the appropriate clippy allow lines instead.